go/ct/spc/specification_test.go

```diff
@@ -240,7 +244,7 @@ func TestSpecification_AtMostOneCodeAtPC(t *testing.T) {
 }
 
 // TODO: re-enable this test when runtime is not an issue anymore.
-// note: before commiting run it locally where timeout is not an issue.
+// note: before committing run it locally where timeout is not an issue.
```

Comment:
has an issue been created to do this ? if so please write the number

Reply:
This is a lint of a misspelled comment.

Reply:
@facuMH you introduced this comment, do you have a follow-up ticket? https://github.com/Fantom-foundation/Tosca/pull/455

Reply:
in my local machine it takes 22.9 seconds. I think this test can be re-enabled and the comment deleted.

Reply:
[#615](https://github.com/Fantom-foundation/Tosca/issues/615)